kubernetes-addons/keda/keda-values.yaml

```diff
@@ -0,0 +1,25 @@
+
+image:
```

Comment:
If we have overrides, let's use the values.yaml file. Otherwise no reason to put default values in here.